testutil/context.go

```diff
@@ -24,3 +24,24 @@ func DefaultContext(key storetypes.StoreKey, tkey storetypes.StoreKey) sdk.Conte
 
 	return ctx
 }
+
+type TestContext struct {
+	Ctx sdk.Context
+	DB  *dbm.MemDB
+	CMS store.CommitMultiStore
+}
+
+func DefaultContextWithDB(key storetypes.StoreKey, tkey storetypes.StoreKey) TestContext {
```

Comment:
sorry one more thing. Why not have this take a `*testing.T` and that way we can do an assertion and avoid the silly panic?